CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 
 * Added `MapView.showsTileSet(with:layerIdentifier:)` and `MapView.setShowsTileSet(_:with:layerIdentifier:)` to provide the ability to show and hide custom tile set identifiers on the map view. ([#3700](https://github.com/mapbox/mapbox-navigation-ios/pull/3700))
 
+### CarPlay
+
+* Added `CarPlayUserInfo` type alias for storing CarPlay-related user information. ([#3709](https://github.com/mapbox/mapbox-navigation-ios/pull/3709))
```

Comment:
When introducing a new type, the changelog entry should ideally focus on where the developer will encounter this type. In this case, we can say that, whenever a `CarPlayDelegate` method is called, the `userInfo` property of any passed-in CarPlay object is of type `CarPlayUserInfo`.